test/bin/stress_testing.sh

```diff
@@ -25,12 +25,58 @@ function run {
 
   if [ "${SSH_HOSTNAME:-}" ]; then
     # shellcheck disable=SC2086
-    ssh -o LogLevel=ERROR ${SSH_PORT_PARAM:-} ${SSH_PKEY_PARAM:-} ${SSH_USER_PARAM:-}${SSH_HOSTNAME} "$@"
+    ssh -o StrictHostKeyChecking=no -o LogLevel=ERROR ${SSH_PORT_PARAM:-} ${SSH_PKEY_PARAM:-} ${SSH_USER_PARAM:-}${SSH_HOSTNAME} "$@"
   else
     "$@"
   fi
 }
 
+function reboot_and_wait {
+  PREV_BOOT_ID=$(run cat /proc/sys/kernel/random/boot_id)
+  run sudo reboot &
+  sleep 10
+  TIMEOUT_LIMIT=$(( SECONDS + 300 ))
+  while true; do
+    NEW_BOOT_ID=$(run cat /proc/sys/kernel/random/boot_id)
```

Comment:
A variable in the loop cannot be read-only
```suggestion
    local NEW_BOOT_ID
    NEW_BOOT_ID=$(run cat /proc/sys/kernel/random/boot_id)
```